src/frontend/apps/lti_site/components/VideoCreateForm/index.tsx

```diff
@@ -380,9 +380,11 @@ const VideoCreateFormStep3 = ({ onNextStep }: VideoCreateFormStep3Props) => {
         <FormattedMessage {...messages.step3Message} />
       </Text>
       <Box align="end">
-        <Button primary onClick={() => onNextStep()}>
-          Create another video
-        </Button>
+        <Button
```

Comment:
in your commit message (commit `bug(frontend) button without label bug `):

`s/When a Grommet Button had not label property/When a Grommet Button has no label property/`